addons/project/static/src/project_sharing/views/form/project_sharing_form_renderer.js

```diff
@@ -1,11 +1,12 @@
 /** @odoo-module */
 
```

Comment:
Since you alter that file, may I ask you to remove those lines since `@odoo-module` is no longer needed? 🥺 

Thanks in advance! 💪🏼